### PR TITLE
Move bats tests to upstream

### PR DESCRIPTION
**Pull request by Stell0**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+    # Install packages needed on runner
+    - name: Install required packages
+      run: sudo apt-get install -y apache2 libapache2-mod-xsendfile php-fpm bats
+
+
+    # download and explode tancredi tarball
+    - name: explode Tancredi
+      run: |
+        wget $(grep Source1 nethserver-tancredi.spec | rev | awk '{print $1}' | rev)
```
Comment (DavidePrincipi) [inline .github/workflows/test.yml]:
```suggestion
        wget $(awk '/^Source1/ {print $2}' nethserver-tancredi.spec)
```

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        echo 'XSendFilePath /var/lib/tancredi/data/firmware' | sudo tee -a /etc/apache2/sites-available/tancredi.conf
+        sudo a2ensite tancredi.conf
+        sudo a2dissite 000-default.conf
+        sudo a2enmod proxy_fcgi
+        sudo systemctl restart apache2
+
+    - name: run bats
+      run: |
+        cd test
+        sudo ./run.sh
```
Comment (DavidePrincipi) [inline .github/workflows/test.yml]: Why sudo? Could you set /etc/tancredi.conf as world writable?

File: test/bats/05_auth.bats [hunk 3]
```diff
@@ -25,6 +25,10 @@ setup () {
 }
 
 @test "Authenticated GET /models" {
+    if [[ ! -d /var/lib/nethserver ]] ; then
+        # Not NethServer, skip authentication test
```
Comment (DavidePrincipi) [inline test/bats/05_auth.bats]: Not NethServer: let's move tests for upstream code to the upstream repository ;)

**Review by DavidePrincipi — COMMENTED** [review]
I propose to split the test suite into 

- tests for upstream code, to be moved to the tancredi repo
- test for NethVoice (auth plugin) to be left here

File: .github/workflows/test.yml (hunk 2 above)
Comment (Stell0) [reply .github/workflows/test.yml]: nope, /etc isn't word writable

**Review by DavidePrincipi — APPROVED** [review]

**Pull request merged**